packages/dds/tree/src/feature-libraries/flex-tree/context.ts

```diff
@@ -75,19 +81,31 @@ export class Context implements FlexTreeContext, IDisposable {
 	private disposed = false;
 
 	/**
-	 * @param forest - the Forest
-	 * @param editor - an editor that makes changes to the forest.
+	 * The forest from the {@link checkout}.
+	 */
+	public readonly forest: IForestSubscription;
+
+	/**
+	 * An editor that makes changes to the {@link forest}.
+	 */
+	public readonly editor: IDefaultEditBuilder;
+
+	/**
+	 * @param schema -
+	 * @param checkout -
```

Comment:
Phrasing for the param docs welcome, I'm still figuring out a good way to describe the checkout that isn't "the checkout object" 😄 .

Reply:
I'd be okay if you just said, "the checkout" :D